connection.go

```diff
@@ -426,8 +431,13 @@ var newClientConnection = func(
 		MaxAckDelay:                    protocol.MaxAckDelayInclGranularity,
 		AckDelayExponent:               protocol.AckDelayExponent,
 		DisableActiveMigration:         true,
-		ActiveConnectionIDLimit:        protocol.MaxActiveConnectionIDs,
-		InitialSourceConnectionID:      srcConnID,
+		// For interoperability with quic-go versions before May 2023, this value must not be set to a value
+		// different from protocol.DefaultActiveConnectionIDs.
```

Comment:
should this be 
`MUST be set to a value different from protocol.DefaultActiveConnectionIDLimit`

Reply:
You’re right. Good catch!